bin/deploy

```diff
@@ -47,15 +81,20 @@ test -f "$LSSTSW/miniconda/.deployed" || ( # Anaconda
 
     (
         # Install packages on which the stack is know to depend
-        # Note: it's not clear if agreement was reached to use all of these,
-        #       or they crept in by accident.
 
-        export PATH="$LSSTSW/miniconda/bin:$PATH"
         # XXX the list of conda packages should be kept in sync with
         # https://github.com/lsst/miniconda/blob/master/ups/eupspkg.cfg.sh
         # The lsstsw list of packages should be considered authoritative
-        conda install --yes numpy scipy matplotlib requests cython sqlalchemy astropy pandas
-        pip install stsci.distutils
+        export PATH="$LSSTSW/miniconda/bin:$PATH"
+
+        if [[ $BLEED_DEPLOY == true ]]; then
+            conda install --yes nomkl numpy scipy matplotlib requests cython sqlalchemy astropy pandas
+            pip install stsci.distutils
```

Comment:
I will remove it since the dep is tracked in the table files now.  However, I will note that I dislike the behavior of the "vendored" EUPS python packages as setuptools downloads deps from pypi and exactly what is retrieved could change based on which packages are setup n the env.  Also, the dep versions are free floating.